.ci/setup_env_rhel.sh

```diff
@@ -11,11 +11,14 @@ cidir=$(dirname "$0")
 source "/etc/os-release" || "source /usr/lib/os-release"
 source "${cidir}/lib.sh"
 
-echo "Get repo for perl-IPC-Run"
-sudo -E yum-config-manager --enable rhui-rhel-7-server-rhui-optional-rpms
+if [ "$(echo "${VERSION_ID} < 8.0" | bc -q)" == "1" ]; then
+	echo "Get repo for perl-IPC-Run"
+	sudo -E yum-config-manager --enable rhui-rhel-7-server-rhui-optional-rpms
```

Comment:
since this is rh7 specific, I think you should change the if condition to 

if [ "${major_version}" == "7" ]; then